Assets/VRTK/SDK/Base/SDK_BaseHeadset.cs

```diff
@@ -71,7 +71,7 @@ protected Transform GetSDKManagerHeadset()
             var sdkManager = VRTK_SDKManager.instance;
             if (sdkManager != null && sdkManager.actualHeadset != null)
             {
-                cachedHeadset = (sdkManager.actualHeadset ? sdkManager.actualHeadset.transform : null);
+                cachedHeadset = (sdkManager.actualHeadset ? sdkManager.actualHeadset.transform : null); //JSL: why not return Camara.main to default to built-in
```

Comment:
I believe we did some checking a while back and `Camera.main` has some overhead on it.

Also, this line change with the comment should be removed. It's not the correct place to start a discussion about features in a pull request ;)